2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -134,39 +134,41 @@ function replset_addr() {
 }
 
 # mongo_remove removes the current MongoDB from the cluster
+# optionally takes identifier of replset to connect and address of server to remove
 function mongo_remove() {
   local host
   # if we cannot determine the IP address of the primary, exit without an error
   # to allow callers to proceed with their logic
-  host="$(replset_addr || true)"
+  host="${1:-$(replset_addr || true)}"
+  local mongo_addr
+  mongo_addr="${2:-$(mongo_addr)}"
```

Comment:
> E.g., it is not easy to provide what you want to remove ($2) without providing the replica set address ($1).

@rhcarvalho You can use `""` as a first argument. I know that this is not elegant, but it is possible.

> Undocumented behavior: is it valid to call mongo_remove to remove something that is not an item in the replica set members?

Yes, IMPOV it is valid but should return the appropriate error.

> When do we want to specify the host to be something that is not replset_addr?

#184 uses only hostnames to addressing.